Dockerfile

```diff
@@ -1,15 +1,27 @@
-FROM ghcr.io/foundry-rs/foundry:nightly
+FROM ghcr.io/foundry-rs/foundry:stable
 
 WORKDIR /app
-
 USER root
 
+# Install system dependencies
+RUN apt-get update && apt-get install -y curl build-essential && rm -rf /var/lib/apt/lists/*
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
```

Comment:
Ideally we could copy a stand-alone binary (and avoid having to add the Rust dependencies here) + limit the files copied into the image even more cc @ruseinov